index.bs

```diff
@@ -1687,6 +1687,31 @@ Open Screen Protocol messages.  An agent should not send extension fields to
 another agent unless that agent advertises an extension capability ID in its
 [=agent-info=] that indicates that it understands the extension fields.
 
+Type Key Backwards Compatibility
+--------------------------------
+
+As messages are modified or extended over time, certain rules must be followed
+to maintain backwards compatibiilty with agents that understand older versions
+of messages.
+
+1.  If a required field is added to or removed from a message, a new type key must
```

Comment:
Do we have any ideas on when the API will be considered "stable enough" to enforce these rules?

Reply:
Probably not until there are agents out in the wild expecting old messages.